src/Typescriptr/Formatters/CommentBlockDecorator.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace Typescriptr.Formatters
+{
+    public class CommentBlockDecorator
```

Comment:
Sorry one more thing - could we please call this something like `GeneratedCodeWarningCommentDecorator`?

Reply:
If you're maybe a bit on the fence about it, I can just move it to the test case? It was only intended to be demonstrative, and I'm sure use cases will vary.

Reply:
sorted 👍